include/openssl/e_os2.h

```diff
@@ -276,6 +276,18 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+/* Format specifier for printing size_t */
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
+      && !defined(__VMS)
+#  define OSSLzu  "zu"
+# else
+#  ifdef SIXTY_FOUR_BIT_LONG
+#   define OSSLzu PRIu64
```

Comment:
Wait, hold on.  *We* define `PRIu64` unconditionally if it isn't already defined, just a few lines up.  So the C99 comment isn't very relevant.